.github/workflows/puppetfile.yml

```diff
@@ -0,0 +1,73 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Puppetfile checks
+
+# Controls when the workflow will run
+on:
+  workflow_call:
+    inputs:
+      container_image:
+        description: Image to use when running tests
+        default: 'puppet/puppet-dev-tools:2022-03-28-92c7176'
```

Comment:
do they offer a latest that that's reliable or could we use the default ubuntu images and install ruby + ra10ke? That would  probably reduce the maintenance cost a bit. (maybe wait for feedback from @ekohl here)